libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { configureEaseInOutLayoutAnimation } from '../utils/animations';
+import { WINDOW_WIDTH } from '../utils/constants';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+  disabled?: boolean;
+} & FxBoxProps;
+
+const FxProgressBar = ({
+  width,
+  progress,
+  height = 4,
+  total = 100,
+  ...rest
+}: FxProgressBarProps) => {
+  const _width = width || WINDOW_WIDTH;
```

Comment:
I was thinking `width` would be the width of the containing component from the parent screen.

we could put an `onLayoutEvent` on the `FxBox` below and get the width that way.

Reply:
Agreed, autolayout would be better